### ISSUE TITLE
not working properly on one UITextField in UITableViewCell (swift 3)

### ISSUE THREAD

**Issue by KileyCaravella** [michaeltyson/TPKeyboardAvoiding#250]
I posted to stack overflow as well [here](https://stackoverflow.com/questions/45580713/swift-3-tpkeyboardavoidingscrollview-not-working-properly-on-one-uitextfield-in[]). 

I'm having this problem where I have three cells, all created by the same function, but for some reason the third one has issues with scrolling. Not sure if it's this pod (which btw is my favorite pod of all time) but maybe there's some mechanic that's messing with it.

![tpavoidingissue](https://user-images.githubusercontent.com/11179948/29103482-d68c0ebe-7c8d-11e7-96ef-658d85206f8a.gif)

All three of the cells are set up the same:

    guard let cell = tableView.dequeueReusableCell(withIdentifier: SignUp_Constants.DetailsCellIdentifier, for: indexPath) as? DetailsCell else { return UITableViewCell() }

    cell.isUserInteractionEnabled = true
    cell.detailsInputTextField.removeTarget(nil, action: nil, for: .allEvents)
    cell.detailsInputTextField.addTarget(cell, action: #selector(DetailsCell.textFieldDidChange(_:)), for: .editingChanged)

    cell.detailsInputTextField.autocapitalizationType = .none
    cell.detailsInputTextField.spellCheckingType = .default
    cell.detailsInputTextField.isSecureTextEntry = false
    cell.detailsInputTextField.isUserInteractionEnabled = true
    cell.detailsInputTextField.keyboardType = .default

    cell.delegate = self

    var title = SignUp_Constants.getTitle(forCellType: currentSectionView, atRow: indexPath.row)
    cell.setTextFieldText(toValue: signUpDictionary[title] as? String ?? "")        
    cell.setTitleLabel(to: title)

    return cell

Basically, when you click on the textfield inside of the third cell to input information, the view scrolls to the bottom of the screen instead of to the textfield. The other two cells scroll correctly to the textfields. I've used the cell with the same code in many other locations in the app, but for some reason the pod really does not like this particular one. Let me know if you have suggestions or need more information! Thanks in advance!

**Comment by KileyCaravella** [comment]
I added `tableView.scrollToRow(row: x, atIndexPath: indexPath)` and the error went away, but I've never had to use that function in conjunction with this pod. Let me know if you have any ideas as to why this happened!
